### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by seulqxq** [Yikai-Wang/asuka-misato#16]
Hi, I have a question. I couldn't seem to find the loss function used for training the decoder in the paper. Could you please provide the details on the loss function and its configuration?

**Comment by Yikai-Wang** [comment]
We follow the setup in https://github.com/buxiangzhiren/Asymmetric_VQGAN.

**Comment by seulqxq** [comment]
> We follow the setup in https://github.com/buxiangzhiren/Asymmetric_VQGAN.

Okay, thank you very much for your reply!
